src/AccessibilityInsights/Misc/CommandOptions.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using CommandLine;
+
+namespace AccessibilityInsights.Misc
+{
+    public class CommandOptions
```

Comment:
>CommandOptions [](start = 17, length = 14)

We need to ensure that the current command line (i.e., a file is specified as the first parameter) continues to work correctly

Reply:
The file scenario still works correctly